legend-engine-xt-analytics-search-generation/src/main/java/org/finos/legend/engine/generation/SearchDocumentArtifactGenerationExtension.java

```diff
@@ -0,0 +1,97 @@
+/*
+ *  Copyright 2023 Goldman Sachs
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.finos.legend.engine.generation;
+
+import static org.finos.legend.engine.language.pure.compiler.toPureGraph.HelperModelBuilder.getElementFullPath;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.eclipse.collections.impl.list.mutable.FastList;
+import org.finos.legend.engine.language.pure.compiler.toPureGraph.PureModel;
+import org.finos.legend.engine.language.pure.dsl.generation.extension.Artifact;
+import org.finos.legend.engine.language.pure.dsl.generation.extension.ArtifactGenerationExtension;
+import org.finos.legend.engine.protocol.pure.v1.model.context.AlloySDLC;
+import org.finos.legend.engine.protocol.pure.v1.model.context.PureModelContextData;
+import org.finos.legend.pure.generated.Root_meta_analytics_search_metamodel_BaseRootDocument;
+import org.finos.legend.pure.generated.Root_meta_analytics_search_metamodel_ProjectCoordinates;
+import org.finos.legend.pure.generated.Root_meta_analytics_search_metamodel_ProjectCoordinates_Impl;
+import org.finos.legend.pure.generated.core_analytics_search_trans;
+import org.finos.legend.pure.generated.core_pure_protocol_protocol;
+import org.finos.legend.pure.m3.coreinstance.meta.pure.metamodel.PackageableElement;
+import org.slf4j.Logger;
+
+public class SearchDocumentArtifactGenerationExtension implements ArtifactGenerationExtension
+{
+    public final String ROOT_PATH = "searchDocuments";
+
+    private static final Logger LOGGER = org.slf4j.LoggerFactory.getLogger("Alloy Execution Server");
```

Comment:
Is this the right logger?

```suggestion
    private static final Logger LOGGER = org.slf4j.LoggerFactory.getLogger(SearchDocumentArtifactGenerationExtension.class);
```